internal/connect/telemetry.go

```diff
@@ -0,0 +1,32 @@
+package connect
+
+import (
+	"log/slog"
+
+	"github.com/SUSE/telemetry"
+)
+
+func pushTelemetry(content []byte) {
+	telemetryType := telemetry.TelemetryType("SCC-CONNECT-MANDATORY")
+	class := telemetry.MANDATORY_TELEMETRY
+	tags := telemetry.Tags{"mandatory", "scc", "connect"}
```

Comment:
You probably don't need to include these tags, since they are redundant, given that this information is derivable from the telemetry type you have chosen.

It is ok for tags to be left empty by telemetry generating applications; they are intended more for specifying ancillary grouping information, potentially set up by the product registration, e.g. SCC connection style of direct vs RMT vs Cloud BYOS, or maybe to track a customer deployment being an MSP (or something like that) generating uptime tracking info, or anything else that may be useful in the long run...